src/ui/pages/payment-entry-page.svelte

```diff
@@ -167,6 +167,7 @@
           bind:submit={stripeSubmit}
           bind:confirm={stripeConfirm}
           bind:stripeLocale
+          bind:priceBreakdown
```

Comment:
A `bind` should not be needed in this case, we can pass eighter the entire `priceBreakdown` or a new field as we did for `taxCollectionEnabled`